src/components/SearchResults/SearchResultsSort.tsx

```diff
@@ -0,0 +1,51 @@
+import { Select } from "@nypl/design-system-react-components"
+import { sortOptions } from "../../utils/searchUtils"
+
+import type { SearchParams } from "../../types/searchTypes"
+import type { ChangeEvent } from "react"
+
+interface SearchResultsSortProps {
+  pageHasResults: boolean
+  searchParams: SearchParams
+  handleSortChange: (e: ChangeEvent) => Promise<void>
+  mobileOnly?: boolean
+}
+
+/**
+ * The SearchResultsSort component renders a Select element used for sorting Search Results.
+ */
+const SearchResultsSort = ({
+  pageHasResults,
+  searchParams,
+  handleSortChange,
+  mobileOnly = false,
+}: SearchResultsSortProps) => {
+  if (!pageHasResults) return
+  return (
+    <Select
+      name="sort_direction"
+      id={`search-results-sort${mobileOnly && "-mobile"}`}
```

Comment:
not a huge issue, but this id comes up as "search-results-sortfalse-wrapper" in the inspector, which I don't believe was the intention.